src/mlpack/bindings/cli/get_param.hpp

```diff
@@ -51,17 +51,21 @@ T& GetParam(
   // contains the filename.  It's possible we could load empty matrices many
   // times, but I am not bothered by that---it shouldn't be something that
   // happens.
-  typedef std::tuple<T, typename ParameterType<T>::type> TupleType;
+  typedef std::tuple<T, typename ParameterType<T>::type, size_t, size_t> TupleType;
```

Comment:
Of course, if you tried the change I suggested in the other comment, you'd have to change this here, and then get `n_rows` and `n_cols` as something like `std::get<0>(std::get<1>(tuple));`